packages/react/src/modules/guide/components/Card/Card.tsx

```diff
@@ -170,47 +196,70 @@ const DefaultView: React.FC<{
         <Title title={content.title} />
         <Body body={content.body} />
       </Content>
-      <Actions>
-        {content.primary_button && (
-          <PrimaryButton
-            text={content.primary_button.text}
-            action={content.primary_button.action}
+      {content.image && (
+        <a
+          href={
+            isValidHttpUrl(content.image.action)
+              ? content.image.action
+              : undefined
+          }
+          target="_blank"
+        >
+          <Img
+            src={content.image.url}
+            alt={content.image.alt}
             onClick={(e) => {
-              if (onButtonClick) {
-                const { text, action } = content.primary_button!;
-                onButtonClick(e, { name: "primary_button", text, action });
+              if (onImageClick) {
+                onImageClick(e, content.image!);
               }
             }}
           />
-        )}
-
-        {content.secondary_button && (
-          <SecondaryButton
-            text={content.secondary_button.text}
-            action={content.secondary_button.action}
-            onClick={(e) => {
-              if (onButtonClick) {
-                const { text, action } = content.secondary_button!;
-                onButtonClick(e, { name: "secondary_button", text, action });
-              }
-            }}
-          />
-        )}
-      </Actions>
+        </a>
+      )}
+      {(content.primary_button || content.secondary_button) && (
```

Comment:
Nothing changed with buttons, but right now we are still rendering `<Actions />` even if there are no buttons and this adds extra spacing at the bottom due to the gap attribute in the container. We should only render it if there is at least one button.